bin/create-cluster.sh

```diff
@@ -70,11 +75,14 @@ gcloud container clusters create $GKE_CLUSTER_NAME \
       --enable-autoscaling \
       --enable-autoupgrade \
       --enable-autorepair \
-      --scopes "gke-default" \
-      --enable-cloud-logging --enable-cloud-monitoring \
+      --scopes  "https://www.googleapis.com/auth/cloud-platform" \
+     --enable-cloud-logging --enable-cloud-monitoring \
```

Comment:
The comment says they're not required, so either the comment is incorrect or you can remove this line.